helm/hpcc/templates/eclccserver.yaml

```diff
@@ -68,6 +68,7 @@ data:
     global:
 {{ include "hpcc.generateGlobalConfigMap" $ | indent 6 }}
 {{- if not .useChildProcesses }} 
+{{- $jobNamePrefix := printf "%s-compile-" .name }}
```

Comment:
as per above comment, we discussed changing the name of the pod to include the target (.name) too.